### PR TITLE
TE-1.2 : Added wait for arp util func

### PR DESCRIPTION
**Pull request by cprabha**
Hi,

I have added WaitForArp after pushconfig otg function.

### CODE REVIEW

**Review by dplore — DISMISSED** [review]

**Comment by dplore** [comment]
Please also fix README (yaml path formatting) please.  Thank you!

**Comment by cprabha** [comment]
> Please also fix README (yaml path formatting) please. Thank you!

updated README , Thanks @dplore .

File: feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md [hunk 1]
```diff
@@ -25,3 +25,13 @@ Ensure my MAC entries installed on the DUT are honored and used for routing.
 ## Protocol/RPC Parameter Coverage
 
 N/A
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+rpcs:
```
Comment (dplore) [inline feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md:32]: Need to add 

```
paths:
   /system/mac-address/config/routing-mac:
   /system/mac-address/state/routing-mac:
```

File: feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md [hunk 2]
```diff
@@ -25,3 +25,13 @@ Ensure my MAC entries installed on the DUT are honored and used for routing.
 ## Protocol/RPC Parameter Coverage
 
 N/A
+
```
Comment (dplore) [inline feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md:28]: lines 17-27 can be removed.

Comment (cprabha) [reply feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md]: updated both. please check.

File: feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md (hunk 1 above)
Comment (cprabha) [reply feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/README.md]: updated

**Review by dplore — APPROVED** [review]

**Pull request merged**